src/utils/cache/getSdk.ts

```diff
@@ -77,7 +77,9 @@ export const getSdk = async ({
   let chain: NetworkInput | undefined = undefined;
   try {
     chain = await getChainByChainIdAsync(chainId);
-  } catch (error) {}
+  } catch (error) {
+    console.log("failed to get the chain");
```

Comment:
can you use `logger()` from our utils folder, instead of `console.log()`. and do we need this error log? is it actionable?

Reply:
yeah, this was a debug log that was missed